gluoncv/utils/filesystem.py

```diff
@@ -138,10 +138,13 @@ def try_import_dali():
     try:
         dali = __import__('nvidia.dali', fromlist=['pipeline', 'ops', 'types'])
         dali.Pipeline = dali.pipeline.Pipeline
-    except ImportError:
+    except (ImportError, RuntimeError) as e:
```

Comment:
You can chain exception like if you want to handle them differently

```python
except xxx as e:
    pass
except yyy as e:
    pass

Reply:
I am not sure if we want to duplicate the dali class definition in this way? I could change it into this way if you want.